pkg/operator/cluster/ceph/mon/health.go

```diff
@@ -111,9 +111,10 @@ func (c *Cluster) checkHealth() error {
 			// delete the "timeout" for a mon if the pod is in quorum again
 			if _, ok := c.monTimeoutList[mon.Name]; ok {
 				delete(c.monTimeoutList, mon.Name)
+				logger.Infof("mon %s is back in quorum, removed from mon out timeout list", mon.Name)
 			}
 		} else {
-			logger.Warningf("mon %s NOT found in quorum. %+v", mon.Name, status)
+			logger.Debugf("mon %s NOT found in quorum. %+v", mon.Name, status)
```

Comment:
is this DEBUG level now because we are just going to log at warning level a few lines down anyways? (making this log statement redundant)

Reply:
Not directly, because the full quorum `status` is printed in the debug line.